src/pages/dmt5/_flexibility.js

```diff
@@ -126,8 +138,20 @@ const Flexibility = () => {
             </StyledHeader>
             <Flex mb="4rem" tablet_direction="column" tablet_ai="center" tablet={{ m: '0' }}>
                 {content.map((item, idx) => {
+                    let show_content = false
+
+                    if (item.show_eu !== undefined) {
+                        if(is_eu_country) {
+                            show_content = item.show_eu
+                        } else {
+                            show_content = !item.show_eu
+                        }
+                    } else {
+                        show_content = true
+                    }
+
                     return (
-                        <ClientCard key={idx}>
+                        <ClientCard key={idx} style={{display : show_content ? 'block' : 'none'}}>
```

Comment:
Please pass the styles in as props into the styled component `ClientCard` on line 61 instead of inline styling.